alicorn-expressions.lua

```diff
@@ -343,6 +354,7 @@ local function expression_symbolhandler(args, name)
 	if not front then
 		local ok, val = env:get(name)
```

Comment:
val here must be a binding according to the type annotations (confirmed with Open), yet a few lines below, it is passed directly into `checkable_term.inferrable(val)`, which only accepts inferrables.